olaf/_internals/services/logs.py

```diff
@@ -47,12 +55,6 @@ def on_loop(self):
         self.sleep(0.1)
 
     def on_read_since_boot(self) -> str:
```

Comment:
This callback should be part of `LogServiceHandler` and not `LogsService`. Beyond doing something involving logs they share no common interests. As an added bonus when this callback is part of `LogServiceHandler`, `_logs` no longer needs to be global.